### PR TITLE
feature/crud articles

### PR DESCRIPTION
**Pull request by misterhtmlcss**
- create a model for article
- added article resource
- stubbed out the 7 restful patterns
- created migration to setup db for index render
- stubbed out controller for all 7 restful methods
- added basic index erb file to render dummy data
- added schema from migration
- fixed missing create Articles migration
- added CRUD functionality to Articles
- added article route and validations
- tweaked a migration

### CODE REVIEW

File: app/views/articles/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
```
Comment (jensen) [inline app/views/articles/index.html.erb:1]: Not sure why this was added to the template. The `app/views/layouts/application.html.erb` is the file that contains the `head` and `body` tags. It then yields to the template being rendered.

File: config/routes.rb [hunk 2]
```diff
@@ -1,5 +1,7 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
 
-  root "home#index"
+  root "article#index"
```
Comment (jensen) [inline config/routes.rb:4]: Does this work? I expected to see `articles#index`.

File: db/migrate/20220326114929_add_title_and_body_to_articles.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+class AddTitleAndBodyToArticles < ActiveRecord::Migration[6.1]
+  def change
+    add_column :articles, :title, :string
```
Comment (jensen) [inline db/migrate/20220326114929_add_title_and_body_to_articles.rb:3]: Is there a specific reason not to limit this to one migration?

Comment (misterhtmlcss) [reply db/migrate/20220326114929_add_title_and_body_to_articles.rb:3]: I wanted to practice and I wanted to see what would work and what wouldn't, etc. It's a fair point though.

**Comment by JakeFantin** [comment]
Ugh, why did aaron tag me in thiiiis. I mean, if it's to get me more involved, that's valid. I can't see anything, I'm Ruby blind! Other than the the DB setup failing in the CI. Looks like you resolve that in the next PR or two. I guess I'll check those out. Otherwise it looks like your Rails CRUD tutorial was very similar to mine.

**Pull request merged**